owning-a-smart-contract.md

```diff
@@ -62,160 +64,124 @@ contract Restricted =
 
 ***Note***: keep formatting intact or things will break badly
 
-As you can see this contract exposes two functions - both returning the number it has received as argument. Note that the restricted function is **not** actually restricted yet.
-### Write unit tests to call the methods
+As you can see this contract exposes two functions - both returning the number it has received as an argument. Note that the restricted function is **not** actually restricted yet.
```

Comment:
`public` function should be `entrypoint` https://forum.aeternity.com/t/hosted-compiler-july-scheduled-update/3937

Reply:
Hmm this tutorial is using `forgae` so compiling is done locally as opposed to using the hosted compiler. So until `forgae` has incorporated these new changes, I think it will be broken if I changed it to `entrypoint` now? I might be misunderstanding though.
Thanks!

Reply:
I've made the preemptive commit to accommodate the new changes in the syntax. Not sure when the next version of `forgae` will be released though.